test/AppContainer.dev.test.js

```diff
@@ -927,6 +928,7 @@ describe(`AppContainer (dev)`, () => {
       await Promise.resolve(1)
 
       expect(spy).not.toHaveBeenCalled()
+      expect(ref).toHaveBeenCalledTimes(1)
```

Comment:
When running only this test it worked but when running all it fails. Those things should probably be wrapped in `act` to work properly. I don't have enough context here to make a good judgement how the full test should work with regards to timing.